docs/documentation/release_notes/topics/26_0_0.adoc

```diff
@@ -62,19 +62,21 @@ The `java-keystore` key provider, which allows loading a realm key from an exter
 
 For more information about this subject, see link:{adminguide_link}#realm_keys[Configuring realm keys].
 
+= Deprecating `keycloak` login Theme
+
+The `keycloak` login theme has been deprecated in favour of the new `keycloak.v2`. The `keycloak` login theme will be removed in a future version.
+
+While the `keycloak` login theme remains available for compatibility reasons, it is strongly recommended to switch to `keycoak.v2` as soon as posible.
+
+For all new realms, `keycloak.v2` will be the default login theme.
```

Comment:
This is wrong, the theme will be the default in every realm unless a specific login theme is configured.